ex_3.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Jun 10 08:52:42 2018
+
+@author: amyhaddad
+"""
+
+
+print("This is exercise 3.")
+
+print("What is your favorite quote?", end= ' ')
```

Comment:
Is there a reason you're using the space at the end instead of the normal line break?